include/crocoddyl/core/actions/lqr.hxx

```diff
@@ -101,7 +125,7 @@ void ActionModelLQRTpl<Scalar>::calcDiff(const boost::shared_ptr<ActionDataAbstr
 }
 
 template <typename Scalar>
-boost::shared_ptr<ActionDataAbstractTpl<Scalar> > ActionModelLQRTpl<Scalar>::createData() {
```

Comment:
I did run it (and discussed with Guilhem) -- this is what the formatter caused :-(

Reply:
I.e. the formatter does not consider this to be incorrect / does not fix it

Reply:
@wxmerkt 
Did you use `clang-6`?
This code is not c++98 compatible; thus, it is weird that this is the case.

@nim65s 
Why is now different than before? 
Are we using other setup in the CI?

Reply:
From what I recall we moved to C++14 already since we implicitly had that requirement anyways (https://github.com/loco-3d/crocoddyl/issues/934), so it shouldn't matter if it's not C++98 compatible:

https://github.com/loco-3d/crocoddyl/blob/9e79f80c6d52b2c6c000106570811d449341cad3/CMakeLists.txt#L63